guides/common/modules/proc_installing-the-kernelcare-package-on-managed-hosts.adoc

```diff
@@ -10,9 +10,21 @@ For more information, see xref:KernelCare_Client_{context}[].
 endif::[]
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select one or multiple hosts.
-. From the *Select Action* menu, select *Schedule Remote Job*.
-. In the *Job category* field, select `Katello via Ansible`.
-. In the *Job template* field, select `Install Package - Katello Ansible Default`.
-. In the *Package* field, enter `kernelcare`.
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run job*.
+. Select *Katello* as *Job category* and *Install Package - Katello Script Default* as *Job template*.
+. Select hosts on which you want to run the job.
+ifdef::satellite,orcharhino,katello[]
+You can filter hosts by host names, host collections, host groups, and a search query.
+endif::[]
+ifndef::satellite,orcharhino,katello[]
+You can filter hosts by host names, host groups, and a search query.
+endif::[]
```

Comment:
I think you could delete lines 16-21. The web UI seems clear and self-evident in this step.

Reply:
Removed. I've also removed it from another procedure in a separate commit. Please ACK.